reports/models.py

```diff
@@ -224,6 +224,7 @@ class Test(models.Model):
     # in this case is because we want to be able to compute/predict
     # the primary key of a Test object ourselves in the processor
     # so we can easily do concurrent writes to the database
+    # this is a hash of the repoid, name, testsuite and env
```

Comment:
Interesting, so the id itself will account for the env now 👀

Reply:
i should rename it to `flags_hash` in the comment here as well but yes, because a test is unique on the flags_hash as well, we will include it in the hash